cogs/TFT.py

```diff
@@ -516,7 +590,7 @@ def get_player_tft_rank(self, region_code, summoner):
             color=discord.Colour.blue()
         )
         embed_msg.title = f"Rank info for {summoner}."
-        if ranks_info is []:
+        if ranks_info == []:
             msg = f"{summoner} is Unranked."
             embed_msg.add_field(value=msg)
```

Comment:
I think this needs a `name` arg.

Reply:
Fixed in [5f6746a](https://github.com/hschawe/NeekosHelp/pull/5/commits/5f6746a26a6d68c5ca404366f2df7e56c7822b31)